search-service/src/test/kotlin/com/egm/stellio/search/service/EntityOperationServiceTests.kt

```diff
@@ -308,7 +309,7 @@ class EntityOperationServiceTests {
 
     @Test
     fun `batch delete should return the list of deleted entity ids when deletion is successful`() = runTest {
-        coEvery { entityPayloadService.deleteEntity(any()) } returns Unit.right()
+        coEvery { entityPayloadService.deleteEntity(any()) } returns mockkClass(EntityPayload::class).right()
         coEvery { authorizationService.removeRightsOnEntity(any()) } returns Unit.right()
```

Comment:
In this test, we are testing the batch delete. Why is` entityPayloadService.deleteEntity()` called in the begining ?

Reply:
because this is what is called by `EntityOperationService.delete(...)`